imports.go

```diff
@@ -0,0 +1,28 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package skywalking
+
+import (
+	// base module imports
+	_ "github.com/google/uuid"
+	_ "github.com/pkg/errors"
+
+	// force compile for the agent core copied
+	_ "github.com/apache/skywalking-go/enhance/core"
+	_ "github.com/apache/skywalking-go/enhance/core/tracing"
+)
```

Comment:
So, these are going to be imported to user codes when they do `import _ "github.com/apache/skywalking-go"` at their codes, right?

Reply:
Yes, we need to force import these modules in the `github.com/apache/skywalking-go`, it is used to tell `go build` that these modules need to be loaded in advance. If they are not imported here, errors will occur when the agent core is copied into the project.

Reply:
Could you rename `enhance` to `agent`?

Reply:
Is `reporter` required to add as well?

Reply:
> Could you rename enhance to agent?

Sure.

> Is reporter required to add as well?
 
When registering the Agent, it is necessary to import `reporter`. I will submit other PRs to improve this import file.

Reply:
> When registering the Agent, it is necessary to import reporter. I will submit other PRs to improve this import file.

So, it is just not imported as the codes are not ready. Am I right?

Reply:
Yes.